From 6e9e95dc0fe89a3c52f50e44ff0096a6e65e46a6 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Wed, 20 Dec 2017 10:49:19 -0500
Subject: [PATCH 7/7] device-manager: drop superfluous
 create_pixel_displays_for_renderer call

commit 29e27637694eefc962d53333c729e6cac1c66518 tried to move
create_pixel_displays_for_renderer down a couple of lines, but it
inadvertently copied it instead of moved it.

This commit fixes that.

https://bugs.freedesktop.org/show_bug.cgi?id=104353
---
 src/libply-splash-core/ply-device-manager.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libply-splash-core/ply-device-manager.c b/src/libply-splash-core/ply-device-manager.c
index cf56f4e..fbf4723 100644
--- a/src/libply-splash-core/ply-device-manager.c
+++ b/src/libply-splash-core/ply-device-manager.c
@@ -713,7 +713,6 @@ create_devices_for_terminal_and_renderer_type (ply_device_manager_t *manager,
                 if (manager->keyboard_added_handler != NULL)
                         manager->keyboard_added_handler (manager->event_handler_data, keyboard);
 
-                create_pixel_displays_for_renderer (manager, renderer);
                 ply_hashtable_insert (manager->renderers, strdup (ply_renderer_get_device_name (renderer)), renderer);
                 create_pixel_displays_for_renderer (manager, renderer);
 
-- 
2.15.1

